.circleci/config.yml

```diff
@@ -29,16 +29,43 @@ workflows:
   drafter:
     jobs:
       - lint
-      - test-gcc5.4
-      - test-gcc6
-      - test-gcc7
+      - test-gcc5.4:
+          filters:
```

Comment:
We can do something like the following with YAML references to de-duplicate the tag filter a little bit:

```yaml
aliases:
  - &tag-filter
    filters:
      tags:
        only: /^v.*/
```

Then:

```yaml
- test-gcc6: *tag-filter
- test-gcc7: *tag-filter
```